src/pages/User/AddUser/index.js

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from "react-redux";
+import { Form, message } from 'antd';
+import { compose, pathOr } from 'ramda';
+import PropTypes from 'prop-types';
+
+import {
+  getTypeAccount,
+  getResourcesByTypeAccount,
+  NovoUsuarioService
+} from '../../../services/usuario';
+import buildUser from './userSpec';
+import AddUserContainer from '../../../containers/User/AddUser';
+import PERMISSIONS from '../../../utils/permissions';
+
+const createUserText = 'Usuário cadastrado com sucesso!'
+const unableCreateUserText = 'Não foi possível cadastrar o usuário!'
+const unableSetThatAccountType = 'Não foi possível pegar as permissões desse tipo de conta!'
+
+const successMessage = messageText => {
+  message.success(messageText);
+};
+
+const errorMessage = messageText => {
+  message.error(messageText);
+};
+
+const AddUser = ({
+  allowAddTypeAccount,
+  history,
+}) => {
+  const [allowCustomPermissions, setAllowCustomPermissions] = useState(false);
+  const [form] = Form.useForm();
+  const [shouldRequest, setShouldRequest] = useState(true)
+  const [typeAccounts, setTypeAccounts] = useState([])
+
+  useEffect(() => {
+    if (shouldRequest) {
+      getAllTypeAccount()
+    }
+  }, [
+    shouldRequest,
+  ])
+
+  const handleSubmit = async (formData) => {
+    try {
+      const { status } = await NovoUsuarioService(
+        buildUser({
+          ...formData,
+          allowCustomPermissions,
+        })
+      );
+      if (status === 422) {
+        throw new Error('422 Unprocessable Entity!')
+      }
+      successMessage(createUserText)
```

Comment:
resetar o form após realizar um cadastro 
![image](https://user-images.githubusercontent.com/50411657/101482008-c8825780-3934-11eb-9a76-6cd57701669a.png)
```suggestion
      form.resetFields()
      successMessage(createUserText)
```